.travis.yml

```diff
@@ -9,20 +9,40 @@ env:
     - MINIKUBE_HOME=$HOME
     - MINIKUBE_VER=v0.28.2
     - K8S_VER=1.11.2
-    - KUBEVIRT_VER=v0.8.0-alpha.1
+    - KUBEVIRT_VER=v0.8.0
 
-matrix:
+jobs:
   include:
-  - name: Check syntax of all templates
-    env: TARGET=syntax-tests
-  - name: Functional test of the Fedora template
-    env: TARGET=functional-tests TEST_FUNCTIONAL=fedora28
-  - name: Functional test of the Ubuntu template
-    env: TARGET=functional-tests TEST_FUNCTIONAL=ubuntu1804
-#  - name: Functional test of the OpenSUSE template
-#    env: TARGET=functional-tests TEST_FUNCTIONAL=opensuse15
-  - name: Functional test of the CentOS/RHEL template
-    env: TARGET=functional-tests TEST_FUNCTIONAL=rhel75
+    - stage: "Tests"
```

Comment:
Any resaon for these changes?

if possible then I'd like to delay this part.

Reply:
Reason why i change it is that i didn't find solution how i can add deploy part to this test matrix. If i add deploy part to top level of this config, then it will make deploy in each test. If i add it as a part of this matrix, then it deploys before all tests finish. And per travis documentation, jobs should be used for this case.

Reply:
I agree with Karel here. This seems to be the recommended way per https://docs.travis-ci.com/user/build-stages#build-stages-and-deployments

Reply:
Ok.